src/App.tsx

```diff
@@ -1,16 +1,21 @@
 import { BrowserRouter, Route, Routes } from "react-router-dom";
 import LoginPage from "./pages/login/login";
 import DashboardLayout from "./layout/dashboard-layout/dashboard-layout";
+import ServerEror from "./pages/login/ServerEror";
```

Comment:
ServerEror emas ServerError bo'lishi kerak. Grammatik xato ketibdi

Reply:
aha hozir to'g'rilayman